clr-k8s-examples/create_stack.sh

```diff
@@ -16,10 +16,13 @@ HIGH_POD_COUNT=${HIGH_POD_COUNT:-""}
 
 # versions
 CANAL_VER="${CLRK8S_CANAL_VER:-v3.9}"
+CILIUM_VER="${CLRK8S_CILIUM_VER:-v1.6}"
 K8S_VER="${CLRK8S_K8S_VER:-}"
 ROOK_VER="${CLRK8S_ROOK_VER:-v1.1.1}"
 METRICS_VER="${CLRK8S_METRICS_VER:-v0.3.5}"
 PROMETHEUS_VER="${CLRK8S_PROMETHEUS_VER:-f458e85e5d7675f7bc253072e1b4c8892b51af0f}"
+CNI=${CNI:-"canal"}
```

Comment:
Can you take the CNI external var as CLRK8S_CNI? I would like the same for RUNNER (CLRK8S_RUNNER) but we already have it external to the script as RUNNER. Perhaps we can fix that one in the future as well.

Reply:
If we are making "CLRK8S_RUNNER" as environment variable, then we probably should update it throughout the repo. I can create a PR for the same, if its priority. @jascott1